odoo/cli/scaffold.py

```diff
@@ -5,8 +5,11 @@
 import os
 import re
 import sys
+import lxml
+from markupsafe import Markup
+
+from odoo.addons.base.models.qweb import QWeb
```

Comment:
Reverse this conversion, it's not at all useful, and assuming anyone even uses this subcommand no one will ever bother writing custom templates in qweb, that's just not sensible. Just make the `jinja2` import conditional, and print some sort of error messages if someone uses the command and doesn't have `jinja2` (around the same place as the `cmdargs` check).

Reply:
No, please revert that and don't make the import conditional! There's no need for dropping a perfectly reasonable dependency that may have many specific and necessary uses in other places.

Changing the format used for `mail.template` is done for reasons related to convenience of displaying and editing dynamic templates. That doesn't apply here, and in some other places.

Reply:
@odony Allright so we keep the jinja dependency for the scaffolding ?

I personally didn't like this commit that I introduced so I'm more than happy to revert it.

Reply:
reverted